src/main/java/com/google/neighborgood/servlets/TaskServlet.java

```diff
@@ -130,8 +130,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         }
       }
       out.append("</div>");
-      out.append("<div class='task-content'>")
-          .append((String) entity.getProperty("detail"))
+      out.append(
```

Comment:
I would prefer to have this onclick function dynamically added through the script file. Another suggestion would be to have the whole task div be clickable (sort of like the user tasks in the admin page). I'd like to work on this latter modification since I was planning on doing same changes to the homepage task list anyway, so I can go ahead and change the onclick part as well if you like.

Reply:
Sure. The user_profile page is kind of different from the main page, so it's definitely better if you could help me change the onclick part.

Reply:
Sounds good!